app/views/content/shared/teaching/_bursary_no.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="bursary_no">
+    <p>As a trainee <%= @front_matter["subject"] %> teacher you can get a tuition fee loan and maintenance loan.</p>
```

Comment:
Should this be 'could', can anyone get a loan?

Reply:
I think we should tweak this as Sarah says - we talk about `you may be eligible for....` / `you can apply for....` on https://getintoteaching.education.gov.uk/funding-and-support/tuition-fee-and-maintenance-loans

So I agree that `could` is probably a better option than `can` - or change it further to be more consistent with the language we use on the Student finance page